.prettierrc

```diff
@@ -2,6 +2,5 @@
   "trailingComma": "all",
   "printWidth": 120,
   "semi": false,
-  "singleQuote": true,
-  "jsxBracketSameLine": true
```

Comment:
@Jack-Works ?

Reply:
oops that an error

Reply:
please revert this change. (see develop branch for the latest value)

Reply:
ok